usagov/vote-gov-drupal
# Vote-2825: Migrate Frontend tests to use page objects

## Jira ticket

[Vote-2825](https://cm-jira.usa.gov/browse/VOTE-2825)

## Description

Updating frontend tests to utilize page objects.  The goal of of this is to help with troubleshooting and being able to maintain tests if an update is needed. This also allows us to reuse testing objects in multiple tests or the same if they are reused. 

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. pull down locally
2. cd into `testing` and run `npm run cy:frontEnd` and verify that tests are still passing 

## Checklist for the Developer




## Checklist for the Peer Reviewers
